javascript/builtins/Promise.json

```diff
@@ -5,6 +5,7 @@
         "Promise": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise",
+            "description": "<code>Promise()</code> constructor",
```

Comment:
This should go before `mdn_url`.

Reply:
> This should go before mdn_url.

Okay, but which order should the attributes follow?
This section? https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#the-__compat-object

Reply:
>  See e.g. https://github.com/mdn/browser-compat-data/blob/master/api/AbortController.json#L56

@Elchi3 showed me the AbortController as an example, and in this case "mdn_url" precede "description". should I fix that, too? (maybe in the another PR)

Reply:
fixed at 👉 https://github.com/mdn/browser-compat-data/pull/2303/commits/4c5d136ecaede25172f7df021f8d196b039b236b (rebased)

Reply:
I would say alphabetical order, but we're currently not enforcing that. It's not a big deal to me.

Reply:
👍